doc/swish/db.tex

```diff
@@ -117,12 +117,12 @@
     (immutable stmt) (mutable timestamp)))}\newline
 
   When a SQL string is not found in the cache,
-  \code{PrepareStatement} is used with the
+  \code{osi\_prepare\_statement} is used with the
   \code{current-database} to make a SQLite statement. The raw
   statement handle is stored in a Scheme record:\newline
   \code{(define-record-type statement (fields (immutable handle)
-    (immutable database)))}\newline The \code{statement} record is
-  not registered with a guardian. The statement is finalized using
+    (immutable database)))}\newline
+  The statement is finalized using
```

Comment:
FinalizeStatement => osi\_finalize\_statement
CloseDatabase => osi\_close\_database
BindStatement => osi\_bind\_statement
StepStatement => osi\_step\_statement
ResetStatement => osi\_reset\_statement
GetStatementColumns => osi\_get\_statement\_columns

Thanks for finding these renames I missed when porting from Swish 1!

Reply:
Good idea. Grep even turned up a hit in http.ms. That led to fixes for a few cases where we had `(receive (after n 'timeout) ...)` instead of `(receive (after n (raise 'timeout)) ...)`.

Reply:
Travis pointed out that it no longer makes sense for http.ms to match the platform-specific error code from `uv_fs_open`.